### PR TITLE
Chore: cleanup pubkey usage

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- cleans up pubKey usage across app

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/akasha/src/components/app-routes.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const AppRoutes: React.FC<unknown> = () => {
             }
           />
           <Route
-            path={`${routes[PROFILE_FEED]}/:pubKey`}
+            path={`${routes[PROFILE_FEED]}/:profileId`}
```
Comment (kenshyx) [inline ui/apps/akasha/src/components/app-routes.tsx]: this is actually the `DID` not the profileId, should be renamed accordingly

File: ui/apps/akasha/src/components/profile-feed-page/profile-feed-page.tsx [hunk 2]
```diff
@@ -35,8 +34,8 @@ const ProfileFeedPage = (props: ProfilePageProps) => {
     if (profileState && profileState.name) {
       return profileState.name;
     }
-    return did;
-  }, [profileState, did]);
+    return profileId;
```
Comment (kenshyx) [inline ui/apps/akasha/src/components/profile-feed-page/profile-feed-page.tsx]: the same, this is the DID not the profileId, revert changes on this file

File: ui/hooks/src/use-dev-profile.ts [hunk 3]
```diff
@@ -143,7 +143,7 @@ const verifySignature = async (payload: VerifySignaturePayload) => {
  * ```typescript
  * const verifySignatureMutation = useVerifySignature();
 
- * verifySignatureMutation.mutate({pubKey: 'bbabcbaa243103inr3u2mab3wivqjjq56kiuwcejcenvwzcmjilwnirecba', message: 'original message', signature: 'some signature to be verified'})
+ * verifySignatureMutation.mutate({profileId: 'bbabcbaa243103inr3u2mab3wivqjjq56kiuwcejcenvwzcmjilwnirecba', message: 'original message', signature: 'some signature to be verified'})
```
Comment (kenshyx) [inline ui/hooks/src/use-dev-profile.ts]: hmm, first review please the changes if you use find and replace

File: ui/apps/akasha/src/components/app-routes.tsx (hunk 1 above)
Comment (kenshyx) [reply ui/apps/akasha/src/components/app-routes.tsx]: there is a different value for profileId which is not being used in routing, this leads to confusion

Comment (josenriagu) [reply ui/apps/akasha/src/components/app-routes.tsx]: Okay, I can use did for pubKeys relating to profiles

File: typings/src/ui/navigation.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ export type NavigateToParams = RequireAtLeastOne<NavigationOptions, 'appName' |
 
 export interface ModalNavigationOptions {
   name: string;
-  pubKey?: string;
+  profileId?: string;
```
Comment (kenshyx) [inline typings/src/ui/navigation.ts]: please move the pr into draft mode until all the changes are made

**Pull request merged**